lesson_03/maps/maps.go

```diff
@@ -0,0 +1,38 @@
+package maps
+
+import (
+	"fmt"
+	"sort"
+)
+
+func sortData(data map[int]string) []string {
+	out := []string{}
+	keys := []int{}
+	for k := range data {
+		keys = append(keys, k)
+	}
+
+	sort.Slice(keys, func(i, j int) bool {
```

Comment:
Maybe you should use [`sort.Ints`](https://pkg.go.dev/sort#Ints) here, but this case works too.

Reply:
fixed